Example/Messaging/Sample/Sample/SceneDelegate.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import SwiftUI
+import FirebaseInstanceID
+import FirebaseMessaging
+import FirebaseInstallations
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate, MessagingDelegate {
+  var window: UIWindow?
+  let identity = Identity()
+
+  func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+    // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+    // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+
+    // Get the managed object context from the shared persistent container.
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
```

Comment:
As mentioned in the previous comment, this could be deleted along with the `.environment` bit on line 35.

Reply:
Done.

Reply:
done.